### ISSUE TITLE
Alexa integration not working anymore

### ISSUE THREAD

**Issue by victoroos** [custom-components/alexa_media_player#1801]
**IMPORTANT:  Please search the issues, including closed issues, and the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**
```
Error setting up entry victor_roosjr@hotmail.com - amazon.com for alexa_media
10:09:26 AM – (ERROR) Alexa Media Player (custom integration) - message first occurred at 9:54:52 AM and shows up 2 times
alexalogin.login((<alexapy.alexalogin.AlexaLogin object at 0x7f3f967aead0>,), {'cookies': {'session-id': <Morsel: session-id=132-7997194-5878262>, 'session-id-time': <Morsel: session-id-time=2302059609l>, 'lc-main': <Morsel: lc-main=en_US>, 'ubid-main': <Morsel: ubid-main=130-9990253-8607905>, 'x-main': <Morsel: x-main=aBuFPHbqwcNb1CGHrJ37AI9ZLEAWAjLG>, 'at-main': <Morsel: at-main=Atza|IwEBIBAgRvTCjHpePfvk2hcDAQL5nh7R4fzywWXlwCdgLzi-LbR1u3whg84AQF0v8agSrzRclNel2osXkLGFSUC3nH4ufskAvjjpnjKUWd-LjWLnDvVUiZWSORhIbMuYiKAVsh8qm6O4e1ugF-iy-0OqaFzTEgUaUhITpoItqVq_lE7cWAUB9ZsV3FWfsHsfz7oVrX4DsGHGxDlpNNqrECyHOz75yddnr45A6KV57OIbHJxeCQ>, 'sess-at-main': <Morsel: sess-at-main="bW3Ni72L4o0nKnqUWj5BX7WtDDO5EL2C2Xv15/46P6I=">, 'session-token': <Morsel: session-token=qOU5Rug2z0M2DGdPD6T5XthpWtjhgQeg7dAU9umdysiqYwIm46xWQmdR9HGOsv8D9M3nRCGBTU0TX8M9HNk9WUiXsjnVy805V//1/XbYBvYrf4F1fiiphZ/QtsVwWA/Dv58z2/0Xajh2boz08xNYImAilymtGH78rVVatHRZy7HFbjFSuS81hU7eDGXUnljZxoLXpAYZxxsDFOVGCLLRK6lnDVj6GBmPs5YtqnlYRQlZqysRmpcgQuZ449KPYtQc>, 'csrf': <Morsel: csrf=-801440865>}}): An error occured accessing AlexaAPI: An exception of type TimeoutError occurred. Arguments: ()
10:09:26 AM – (WARNING) Alexa Media Player (custom integration) - message first occurred at 9:54:52 AM and shows up 2 times
Waiting on integrations to complete setup: alexa_media, tado
9:54:48 AM – (WARNING) bootstrap.py - message first occurred at 9:50:48 AM and shows up 5 times
```

**To Reproduce**
NOthing

**Expected behavior**
it would, well, work. It did before and nothing changed in my HA

**System details**
 - Home-assistant (version): 
 - alexa_media  (version from `const.py` or HA startup): latest via HACS
 - alexapy (version from `pip show alexapy` or HA startup): alexapy==1.26.4
 - Amazon [2FA is enabled yes

**Logs**
Logs provided in the bug description

```
2022-12-20 10:09:26.459 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry *here was the email adress which is correct ;)* - amazon.com for alexa_media
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 372, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/alexa_media/__init__.py", line 324, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
  File "/usr/local/lib/python3.10/site-packages/alexapy/helpers.py", line 141, in wrapper
    return await func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/alexapy/alexalogin.py", line 578, in login
    if await self.test_loggedin(cookies):
  File "/usr/local/lib/python3.10/site-packages/alexapy/alexalogin.py", line 457, in test_loggedin
    await self.exchange_token_for_cookies()
  File "/usr/local/lib/python3.10/site-packages/alexapy/alexalogin.py", line 926, in exchange_token_for_cookies
    response = await self._session.post(
  File "/usr/local/lib/python3.10/site-packages/alexapy/aiohttp/client.py", line 466, in _request
    with timer:
  File "/usr/local/lib/python3.10/site-packages/alexapy/aiohttp/helpers.py", line 721, in __exit__
    raise asyncio.TimeoutError from None
[home-assistant_2022-12-20T09-11-04.945Z.log](https://github.com/custom-components/alexa_media_player/files/10267032/home-assistant_2022-12-20T09-11-04.945Z.log)
```

**Comment by alandtse** [comment]
Does your HA have access to the internet and to alexa.amazon.com (or whatever region you're at)? It's reporting a timeout error trying to connect to Amazon.
